hackathon/models.py

```diff
@@ -153,6 +158,20 @@ class HackProject(models.Model):
         related_name="hackproject",)
     display_name = models.CharField(default="", max_length=255)
     description = models.TextField(max_length=500)
+    technologies_used = models.CharField(
+        default="", max_length=1024, 
+        help_text=("Add any technologies that were used for this project"))
+    project_image = models.TextField(
+        default="",
+        blank=True,
+        help_text=("Image displayed next to the project on the team's page.")
+    )
+    screenshot = models.TextField(
```

Comment:
Would be cool to have, but are we hosting these images/screenshots somewhere?

Reply:
Yeah, as base64 image in the database as I wanted to avoid linking S3 (or similar to the project).